lib/routing/app_router.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:go_router/go_router.dart';
+import 'package:tune_catcher/feat/home/home_page.dart';
+import 'package:tune_catcher/feat/recorder/recorder_page.dart';
+import 'package:tune_catcher/feat/recording_list/recording_list_page.dart';
+import 'package:tune_catcher/feat/tune_list/tune_list_page.dart';
```

Comment:
I feel good about this I feel like this list as our MVP page set. 
If we can accomplish this with the appropriate backend I feel like we're gonna have a solid, usable app.